airflow/providers/cncf/kubernetes/__init__.py

```diff
@@ -27,7 +27,7 @@
 
 __all__ = ["__version__"]
 
-__version__ = "8.1.1"
+__version__ = "8.1.2"
```

Comment:
This change is not needed here, could you revert it?

Reply:
Yes, reverted

Reply:
Regarding to tests, I couldn't install the env following the contributing guide, tomorrow I will try again